### PR TITLE
Doc Improvements

### PR DESCRIPTION
**Pull request by spa0002**
- Table of contents is now expandable from the sections: Introduction, Search, Visualize, Access, Query, User Data, FAQ
   - rst files added to make expandable table of contents possible
   - index.rst and conf.py updated
   - welcome.ipynb and git_clone_repo.ipynb moved to "intro" directory (notebooks that appear in the Introduction section)
- Graphics added to "faq" section
- Graphic paths changed in git_clone_repo.ipynb 
- Grammar edits for 'faq' section

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -17,6 +17,69 @@ Welcome to the MAAP User Documentation!
   userdata.rst
   faqs.rst
   
+<<<<<<< HEAD
```
Comment (abarciauskas-bgse) [inline docs/source/index.rst:20]: @spa0002 I think this needs to be removed, looks like a merge conflict

File: docs/source/index.rst [hunk 2]
```diff
@@ -17,6 +17,69 @@ Welcome to the MAAP User Documentation!
...
+  faq/get_dps_job_results.ipynb
+  faq/persistent_files.ipynb
+  faq/run_dps_job.ipynb
+  faq/view_registered_algorithms.ipynb
+  faq/copy_earthdata.ipynb
+  faq/import_granules.ipynb
+  faq/rename_workspace.ipynb
+  faq/share_workspace.ipynb
+=======
+>>>>>>> sayers-doc_improvements
```
Comment (abarciauskas-bgse) [inline docs/source/index.rst:82]: same here

**Review by abarciauskas-bgse — CHANGES_REQUESTED** [review]

File: docs/source/index.rst (hunk 1 above)
Comment (spa0002) [reply docs/source/index.rst]: Removed line 20

File: docs/source/index.rst (hunk 2 above)
Comment (spa0002) [reply docs/source/index.rst]: Removed lines 81,82

**Review by abarciauskas-bgse — APPROVED** [review]

**Comment by abarciauskas-bgse** [comment]
@spa0002 I approved these changes but should we include the `git_clone` notebook somewhere? Seems like the git clone instruction is helpful and it appears to be removed in this PR

**Comment by spa0002** [comment]
That and the 'welcome' notebook should be in a section titled Introduction now - 
https://github.com/MAAP-Project/maap-documentation/blob/sayers-doc_improvements/docs/source/intro.rst
https://github.com/MAAP-Project/maap-documentation/tree/sayers-doc_improvements/docs/source/intro

**Comment by abarciauskas-bgse** [comment]
Thanks @spa0002 approved so I think this is good to merge?

**Pull request merged**